vmdb/product/charts/miq_reports/vim_perf_daily.yaml

```diff
@@ -158,6 +162,8 @@ headers:
 - Min % Used
 - Max % Used
 - Trend Max % Used
+- Used Cores
+- Total Cores

```

Comment:
@Fryguy I think so, but there's been a debate on this lately as to if all of the providers are populating these the same.  Ping @blomquisg